lib/client/presence/local-doc-presence.js

```diff
@@ -30,10 +30,20 @@ LocalDocPresence.prototype.submit = function(value, callback) {
     // be null. Let's early return, instead of error since this is a harmless
     // no-op
     if (value === null) return this._callbackOrEmit(null, callback);
-    var error = {
-      code: ERROR_CODE.ERR_DOC_DOES_NOT_EXIST,
-      message: 'Cannot submit presence. Document has not been created'
-    };
+
+    var error = null;
+    if(this._doc._isInHardRollback) {
```

Comment:
I decided to not change `_isInHardRollback` to `isInHardRollback` as i think it should not be used out side of lib. Feels a bit code smell TBH. I am open for suggesstions.

```suggestion
    if(this._doc._isInHardRollback) {
```